src/adaXT/base_model.pyx

```diff
@@ -62,19 +65,19 @@ class BaseModel:
                     self.leaf_builder_class = leaf_builder
                 else:
                     self.leaf_builder_class = LeafBuilderRegression
-            elif tree_type == "LinearRegression":
+            elif tree_type == "Gradient":
```

Comment:
Would we not want a check both for the linear and the new quadratic?

Reply:
I think the Partial_linear criteria is too niche to deserve its own default